plugins/org/src/components/Cards/Group/MembersList/MembersListCard.stories.tsx

```diff
@@ -29,6 +32,16 @@ import { MembersListCard } from './MembersListCard';
 export default {
   title: 'Plugins/Org/Group Members List Card',
   component: MembersListCard,
+  decorators: [
+    (Story: ComponentType<PropsWithChildren<{}>>) =>
+      wrapInTestApp(
+        <div>
+          <Story />
+        </div>,
+        // The "entityRouteRef" is imported from "@backstage/plugin-catalog-react"
```

Comment:
No need to point this out, it's already clear from the code

Reply:
Removed this comment line